inc/fields/class-shortcode-ui-field-post-select.php

```diff
@@ -94,6 +94,12 @@ public function action_wp_ajax_shortcode_ui_post_field() {
 		$requested_shortcode = isset( $_GET['shortcode'] ) ? sanitize_text_field( $_GET['shortcode'] ) : null;
 		$requested_attr      = isset( $_GET['attr'] ) ? sanitize_text_field( $_GET['attr'] ) : null;
 
+		$include = filter_input( INPUT_GET, 'include', FILTER_SANITIZE_NUMBER_INT, FILTER_REQUIRE_ARRAY );
```

Comment:
I'm not sure I'm following what `FILTER_SANITIZE_NUMBER_INT` is doing here, since you map `intval` over the array a couple lines below. Is the sanitize_int filter just necessary to pass the lint rule?

Reply:
Yes this seems like an overzealous linter change really, will revisit this PR and see if it's still relevant.